src/control/cmd/dmg/storage.go

```diff
@@ -84,16 +83,14 @@ type storageScanCmd struct {
 	connectedCmd
 }
 
-// run NVMe and SCM storage and health query on all connected servers
-func storageScan(log logging.Logger, conns client.Connect) {
-	cNvmeScan, cScmScan := conns.StorageScan()
-	log.Infof("NVMe SSD controllers and constituent namespaces:\n%s", cNvmeScan)
-	log.Info(cScmScan.String())
-}
-
-// Execute is run when storageScanCmd activates
+// Execute is run when storageScanCmd activates.
+// Runs NVMe and SCM storage scan on all connected servers.
 func (s *storageScanCmd) Execute(args []string) error {
-	storageScan(s.log, s.conns)
+	req := client.StorageScanReq{NvmeHealth: false}
```

Comment:
Is the plan to make the health stats optional on scan at some point? I was under the impression the health stats were only going to be queried by the nvme-health command.

Reply:
Yes, that is the goal. We're still working on the details of how these tools will display output, but my preference would be for the health stats to only be displayed on scan if values are out of their normal ranges.

Reply:
the flag in the storage scan request signifies whether or not the output contains the health information, the nvme-health subcommand calls StorageScan with the flag in the request set to true